<launch>


	<param name="molar_point_cloud_topic" type="string" value="/head_xtion/depth_registered/points" />
	<param name="molar_rgb_img_topic" type="string" value="/head_xtion/rgb/image_rect" />
	<param name="molar_depth_img_topic" type="string" value="/head_xtion/depth/image" />
	<param name="molar_camera_info_topic" type="string" value="/head_xtion/rgb/camera_info" />
	<param name="molar_robot_pose_topic" type="string" value="/robot_pose" />
	<param name="molar_tf_topic" type="string" value="/tf" />


	<node name="molar_tools" pkg="molar_tools" respawn="true" type="core.py"  output="screen"/>
	<node name="molar_controller" pkg="molar_controller" respawn="true" type="core.py"  output="screen"/>

	<node name="molar_sensor_layer" pkg="sensor_layer" respawn="true" type="core.py"  output="screen"/>
  <node name="molar_segment_layer" pkg="segment_layer" respawn="true" type="core.py"  output="screen"/>
	<node name="molar_processing_layer" pkg="processing_layer" respawn="true" type="core.py"  output="screen"/>
	<node name="molar_filter_layer" pkg="filter_layer" respawn="true" type="core.py"  output="screen"/>
	<node name="molar_tracking_layer" pkg="tracking_layer" respawn="true" type="core.py"  output="screen"/>



</launch>
